public/app/core/hooks/useStoredTablePageSize.test.tsx

```diff
@@ -0,0 +1,67 @@
+import { mount } from 'enzyme';
+import React, { FC } from 'react';
+import { useStoredTablePageSize } from './useStoredTablePageSize';
+import { PAGE_SIZES } from '../constants';
+
+const TABLE_HASH = 'test-hash';
+const TABLE_HASH_STORAGE_ID = `${TABLE_HASH}-table-page-size`;
+const DEFAULT_VALUE = PAGE_SIZES[0].value;
+
+const TestComponent: FC = () => {
+  const [pageSize, setPageSize] = useStoredTablePageSize(TABLE_HASH);
+
+  return (
+    <>
+      <span>{pageSize}</span>
+      <input type="number" value={pageSize} onChange={e => setPageSize(+e.target.value)} />
+    </>
+  );
+};
+
+const getDataFromLocalStorage = (): number => {
+  return +localStorage.getItem(TABLE_HASH_STORAGE_ID);
```

Comment:
```suggestion
  return parseInt(localStorage.getItem(TABLE_HASH_STORAGE_ID));
```

Please use an explicit conversion

Reply:
👍